.golangci.yml

```diff
@@ -8,12 +10,6 @@ linters:
     - misspell
     - rowserrcheck
     - errorlint
-  disable:
-    # These are all considered deprecated: https://github.com/golangci/golangci-lint/issues/1841
-    - deadcode
-    - structcheck
-    - unused
-    - varcheck
```

Comment:
Three have been removed and don't need to be disabled. `unused` is actually the replacement, and seems helpful if working - though may flag false positives... :detective: